src/ui/modal/modalAutoGenerateThumbnail/index.js

```diff
@@ -1,11 +1,12 @@
 import { connect } from 'react-redux';
 import { doHideModal } from 'redux/actions/app';
 import { doToast, doUploadThumbnail } from 'lbry-redux';
+import fs from 'fs';
 import ModalAutoGenerateThumbnail from './view';
 
 const perform = dispatch => ({
   closeModal: () => dispatch(doHideModal()),
-  upload: buffer => dispatch(doUploadThumbnail(null, buffer)),
+  upload: buffer => dispatch(doUploadThumbnail(null, buffer, null, fs)),
```

Comment:
should these 3rd and 4th params be re-ordered? (meh)

Reply:
The third param is the android equivalent of `fs`